test/integration-test.py

```diff
@@ -265,6 +265,28 @@ def expect(target_time, num):
     expect(now, 0)
     expect(after_grace_period, 1)
 
+def run_certificates_per_name_test():
+    try:
+        handle = subprocess.Popen(
+            '''node test.js --email %s --domains %s --challType %s''' %
```

Comment:
There's no need to specify `challType` here since that's not what we're testing.